plugin_interpreter/src/rethink_interface.py

```diff
@@ -85,6 +85,39 @@ def _get_next_job(self, plugin_name):
             self.plugin_queue.put(new_job)
         except rethinkdb.ReqlCursorEmpty:
            self.plugin_queue.put(None)
+    
+    def _send_output(self, output_data):
+        """sends the plugin's output message to the Outputs table
+        
+        Arguments:
+            output_data {Tuple (str,str)} -- tuple containing the id of the job
+            and the output to add to the table
+        """
+        #get the job corresponding to this output
+        try:
+            output_job = rethinkdb.db("Brain").table("Jobs").get(output_data[0]).run(self.rethink_connection)
+        except rethinkdb.ReqlDriverError:
```

Comment:
You might want to pass the exception to the logger so you can see what actually happened, maybe something like:

```
except rethinkdb.ReqlDriverError as ex:
    self.logger.send([
        ...
        "".join(("Error accessing 'Jobs' table: ", str(ex)))
    ])
```